concent_api/core/tasks.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from celery import shared_task
+
+from conductor.models import VerificationRequest
+from verifier.tasks import blender_verification_order
+
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task
+def upload_finished(subtask_id: str):  # pylint: disable=unused-argument
+    pass
+
+
+@shared_task
+def upload_acknowledged(subtask_id: str):
+    assert isinstance(subtask_id, str)
```

Comment:
This should not be defined in `core`. It's a notification for Conductor that Concent has received `upload_finished` message and that it agrees to start verification. It should be processed by `conductor`.

Reply:
:+1:  done